### PR TITLE
W-13484331-apikeysonlyvisibleoncreation-se

### PR DESCRIPTION
**Pull request by sebermuley**
User and bot API keys are no longer visible after creation. They need to be copied during creation.
-> Adapted and enhanced the docs.

### CODE REVIEW

File: rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc [hunk 1]
```diff
@@ -1,6 +1,8 @@
-= Connecting to MuleSoft Composer
+= Connecting to RPA Manager from Salesforce Products
```
Comment (jyotirmi) [inline rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc:1]: I think the wording "Connecting to RPA Manager" seems incorrect to me. How about - "Connecting RPA Processes from Salesforce Products"? I feel connecting to RPA manager may be interpreted differently by users.

Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc]: Thank you. That is better.

File: rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc [hunk 2]
```diff
@@ -13,35 +15,62 @@ User API keys are required for the integration of RPA processes in a MuleSoft Co
...
+Select yourself. Your identity and permissions are used for the connection from another Salesforce product to RPA Manager.
 * *Expiration Date*:
 +
 Select an expiration date for the user API key from the calendar.
 . Click *OK*.
+. In the window *Key Created!*, click *Copy to clipboard*.
+. Copy the API key to a secure place.
+. Click *Close*.
+
+Copy your saved API key into the configuration settings of the Salesforce products you want to integrate the RPA processes.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc:47]:
```suggestion
Copy your saved API key into the configuration settings of the Salesforce products for which you want to integrate the RPA processes.
```

File: rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc [hunk 3]
```diff
@@ -13,35 +15,62 @@ User API keys are required for the integration of RPA processes in a MuleSoft Co
...
+. Click the *Edit* icon image:edit-icon.png["pen-to-paper symbol",1.5%,1.5%] on the panel of the user API key to edit.
+. Edit the form *Create User API Key*:
++
+For an explanation of the properties, see <<form-createuserapikey, *Create a User API Key*>>.
+. Click *Save*.
+
+== Delete a User API Key
 
-== Copy an API Key to the Clipboard
+Delete a user API key to revoke REST API access to RPA Manager from the user. Integrated processes using the deleted API key stop working.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc:64]:
```suggestion
Delete a user API key to revoke REST API access from the user to RPA Manager. Integrated processes that use the deleted API key stop working.
```

File: rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc [hunk 4]
```diff
@@ -1,6 +1,8 @@
-= Connecting to MuleSoft Composer
+= Connecting RPA Processes from Salesforce Products
 
-User API keys are required for the integration of RPA processes in a MuleSoft Composer flow. Use your API key to configure an API connector in MuleSoft Composer.
+RPA Manager requires user API keys for the integration of RPA processes into other Salesforce products. For example, use your API key to configure an RPA connector in a MuleSoft Composer flow.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc:3]:
```suggestion
RPA Manager requires user API keys for the integration of RPA processes into other Salesforce products. For example, you can use your API key to configure an RPA connector in a MuleSoft Composer flow.
```

File: rpa-manager/modules/ROOT/pages/botmanagement-support.adoc [hunk 5]
```diff
@@ -21,20 +21,31 @@ If a bot only needs to work at specific times, you assign it service times.
...
 
-Edit an API key to change its name or description.
+Edit an API key to change its name or description. If you lose an API key, create a new one.
 
-. In the *API Key* view of the *Bot Management* module, click the *Edit* icon image:edit-icon.png["pen-to-paper symbol",1.5%,1.5%] in the table row of the API key to edit.
+To edit an API key:
+
+. Open the *API Key* view of the *Bot Management* module.
+. Click the *Edit* icon image:edit-icon.png["pen-to-paper symbol",1.5%,1.5%] in the table row of the API key to edit.
 . In the window *Edit Bot API Key*, change name or description and click *OK*.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/botmanagement-support.adoc:49]: the name or description

File: rpa-manager/modules/ROOT/pages/botmanagement-support.adoc [hunk 6]
```diff
@@ -43,14 +54,11 @@ Remove API keys you no longer need.
 
 When you delete an API key, the system does not check whether the key is assigned to a bot. If you have deleted an API key that is in use, the bot can no longer connect to RPA Manager until a new API key is assigned.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/botmanagement-support.adoc:55]: If you deleted an API key...

**Review by dhirsch1 — APPROVED** [review]

File: rpa-manager/modules/ROOT/pages/botmanagement-support.adoc (hunk 6 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/botmanagement-support.adoc:55]: = at the time you delete

File: rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc (hunk 2 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc:47]: into which?

File: rpa-manager/modules/ROOT/nav.adoc [hunk 7]
```diff
@@ -5,7 +5,7 @@
 * xref:usermanagement-overview.adoc[User Management]
 ** xref:usermanagement-manage.adoc[Managing Users]
 ** xref:usermanagement-assemble.adoc[Assembling User Groups]
-** xref:usermanagement-connect.adoc[Connecting to MuleSoft Composer]
+** xref:usermanagement-connect.adoc[Connecting RPA Processes from Salesforce Products]
```
[the remark this answers is not in the document]
Comment (sebermuley) [reply rpa-manager/modules/ROOT/nav.adoc:8]: The latter, I guess...
We can integrate an RPA process in the form of a Salesforce ? / Mulesoft RPA Connector into a Salesforce / Composer flow. 
When the flow reaches the RPA process step, it needs to connect to RPA Manager using a user's API key.
(The permission set of the user determines if the process may be executed.)

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**